exporter/jaegerexporter/config.go

```diff
@@ -29,3 +29,10 @@ type Config struct {
 
 	configgrpc.GRPCClientSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 }
+
+var _ config.Exporter = (*Config)(nil)
+
+// Validate checks if the exporter configuration is valid
+func (cfg *Config) Validate() error {
```

Comment:
We could move https://github.com/open-telemetry/opentelemetry-collector/blob/4a6a25e033f6fce1bca027af5fc354180842cbc7/exporter/jaegerexporter/factory.go#L60-L66 into the validation function.